pkg/repo/repo.go

```diff
@@ -55,6 +56,12 @@ func NewRepoFile() *RepoFile {
 func LoadRepositoriesFile(path string) (*RepoFile, error) {
 	b, err := ioutil.ReadFile(path)
 	if err != nil {
+		if strings.HasSuffix(err.Error(), "no such file or directory") {
```

Comment:
It'd be better to change this to [`if os.IsNotExist(err) {`](https://golang.org/pkg/os/#IsNotExist)

Reply:
Done

Reply:
Doh wait, I don't think you mean it like that :grimacing: Let me try one more time :joy:

Reply:
All right, think I got it :sweat_smile: